app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -42,25 +42,26 @@
     text: "Towards the end of the internship, I was able to teach parts of lessons that I had planned, and I really started to find my feet at the front of the classroom.",
     name: "Ben, former teaching intern",
     background: "grey",
-    large: false
+    large: true
   ) %>
 </section>
 
-<h2> What you'll cover</h2>
+<h2> What you'll experience</h2>
 
-<p>You'll get a feel for school life, which could include:</p>
+<p>You'll get a feel for what life as a teacher is like. Your internship will give you an opportunity to:</p>
   <ul>
-     <li>shadowing experienced teachers and observing lessons</li>
-     <li>jointly planning and delivering lessons</li>
+     <li>shadow experienced teachers and observe lessons</li>
+     <li>jointly plan and teach lessons</li>
      <li>one-to-one guidance from outstanding classroom teachers</li>
-     <li>opportunities to network with qualified subject specialists</li>
+     <li>get one-to-one guidance from outstanding classroom teachers and subject specialists</li>
+     <li>see what goes into being a teacher outside of the classroom</li>
```

Comment:
@philspray I think this line is really weird - what does it mean? Think this is worth discussing with the internships team and rewording